core/crypto/ecdsa_types.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_CRYPTO_ECDSA_TYPES_HPP
+#define KAGOME_CORE_CRYPTO_ECDSA_TYPES_HPP
+
+#include <cstdint>
+#include "libp2p/crypto/ecdsa_types.hpp"
+
+#include "common/blob.hpp"
+
+namespace kagome::crypto {
+  using namespace libp2p::crypto::ecdsa;
+
+  namespace constants::ecdsa {
+    enum {
```

Comment:
Why are not constants instead enum?

Reply:
Copy-Paste from ed25519

Reply:
Think to use constants instead them. Using of anonymous enums does not make sense here, only makes questions.